content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -89,6 +89,7 @@ different Kubernetes components.
 | `DevicePlugins` | `false` | Alpha | 1.8 | 1.9 |
 | `DevicePlugins` | `true` | Beta | 1.10 | |
 | `DisableAcceleratorUsageMetrics` | `false` | Alpha | 1.19 | 1.20 |
+| `DisableAcceleratorUsageMetrics` | `true` | Beta | 1.20 | 1.22 |
```

Comment:
I believe we try to avoid adding future releases into the table since it cannot be guaranteed. @sftim could you confirm?

Reply:
I think it depends on the guarantee - for a deprecation, we might have a clear plan to update this feature gate in a specific future release.
@RenaudWasTaken do you have that context?